lib/msf/core/exploit.rb

```diff
@@ -1482,26 +1482,22 @@ def handle_exception e
       when Rex::ConnectionError
         self.fail_reason = Msf::Exploit::Failure::Unreachable
         self.print_error("Exploit failed [#{self.fail_reason}]: #{msg}")
-        elog("Exploit failed (#{self.refname}): #{msg}", 'core', LEV_0)
-        dlog("Call stack:\n#{e.backtrace.join("\n")}", 'core', LEV_3)
+        elog("Exploit failed (#{self.refname}): #{msg}", error:e)
```

Comment:
we're losing a good amount of debug logging here, why did you decide to take it out?